clang/docs/ReleaseNotes.rst

```diff
@@ -265,6 +265,9 @@ related warnings within the method body.
   ``format_matches`` accepts an example valid format string as its third
   argument. For more information, see the Clang attributes documentation.
 
+- Format string checking now supports the compile-time evaluation of format
```

Comment:
It may help users to understand the improvement if there's a small code example showing what wasn't checked and is now correctly caught. WDYT?